plugins/services/src/js/routes/services.js

```diff
@@ -14,8 +14,7 @@ import TaskDetailBreadcrumb from '../pages/nodes/breadcrumbs/TaskDetailBreadcrum
 import VolumeDetail from '../components/VolumeDetail';
 import VolumeTable from '../components/VolumeTable';
 
-function buildServiceCrumbs(router) {
-  let {id} = router.getCurrentParams();
+function buildServiceCrumbs({id}) {
```

Comment:
Then we'd better to delete everything related to Breadcrumbs and I don't want that here... otherwise it's deleting one method and keeping 500 lines there. That'd be unfair to this method :)